nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/check_samplesheet.py

```diff
@@ -82,10 +83,7 @@ def check_samplesheet(file_in,file_out):
             if sample not in sample_run_dict:
                 sample_run_dict[sample] = [sample_info]
             else:
-                if sample_info in sample_run_dict[sample]:
-                    print_error("Samplesheet contains duplicate rows!",line)
-                else:
-                    sample_run_dict[sample].append(sample_info)
```

Comment:
Here I was thinking maybe it makes more sense to ensure that all samples have unique sample identifiers?